config/stands.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
It would be easier to debug if those IDs would go from 0..n, but @Noarkhh will need to change this anyway, so you can leave it